### PR TITLE
Store the JSON response dictionary in model objects.

### PR DESCRIPTION
**Pull request by RicoYao**
This could be helpful for developers that want to persist the raw response data in their apps.

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @RicoYao, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by RicoYao** [comment]
CLA signed

**Comment by boxcla** [comment]
Verified that @RicoYao has just signed the CLA. Thanks, and we look forward to your contribution.

File: BoxContentSDK/BoxContentSDK/Models/BOXModel.m [hunk 1]
```diff
@@ -15,6 +15,8 @@ @implementation BOXModel
 - (instancetype)initWithJSON:(NSDictionary *)JSONResponse
 {
     if (self = [super init]) {
+        self.JSONResponse = JSONResponse;
```
Comment (boxtcarpel) [inline BoxContentSDK/BoxContentSDK/Models/BOXModel.m]: Minor - naming: JSONData?

In the context of coming from a network request, json response works, but once you're no longer in that state, it's just the JSON data the represents your model.

**Comment by boxtcarpel** [comment]
+2 LGTM

:golf:

File: BoxContentSDK/BoxContentSDK/Models/BOXModel.m (hunk 1 above)
Comment (RicoYao) [inline BoxContentSDK/BoxContentSDK/Models/BOXModel.m]: Done.

**Pull request merged**